ec-service/src/test/java/org/dcsa/endorsementchain/datafactories/EblEnvelopeTODataFactory.java

```diff
@@ -26,4 +28,10 @@ public List<EblEnvelopeTO> eblEnvelopeTOList() {
 
     return List.of(EblEnvelopeTODataFactory.eblEnvelopeTO(), secondEblEnvelope);
   }
+
+  @SneakyThrows
+  public String rawEblEnvelope() {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
This is not using the same object mapper as the injected one (which means it will not use the same configuration)

Reply:
moved the serialization from the datafactory to the unittests using them